### PR TITLE
Fixed media queries, (kind of), lots of linting, got the pages to reload, (kind of)

### PR DESCRIPTION
**Pull request by dylanavery720**
Hey @StephanieEA @bekahlundy , changed BrowserHistory to HashHistory, its kind of hacky  I think but allows us to reload page and hit back and forth. Also we got mr burns centered on the mobile app, has to use the !important in css to make that happen. I deleted footer and a bunch of other junk and did a lot of linting. 

Also fixed a few other errors in consoel. CSS needs some more polishing, and we need to deploy to firebase. Let me know if everything looks alright! 

Dylan

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -11,8 +11,10 @@
     "eol-last": [0],
     "import/no-unresolved": [0],
     "no-mixed-requires": [0],
+    "class-methods-use-this": [0],
```
Comment (stephanieea) [inline .eslintrc:14]: Sweet!

File: lib/components/AccountSetup.js [hunk 2]
```diff
@@ -34,9 +33,9 @@ const { keys, updateMonthly, user, itemValue, budgetItem, submission, updateValu
             onClick={submission}
             text='Submit'/>
 
-             <p><Link to='/public/budgets' aria-label='click here'><div className='burns-sign-2'></div></Link></p>
+             <Link to='/budgets' aria-label='click here'><div className='burns-sign-2'></div></Link>
```
Comment (stephanieea) [inline lib/components/AccountSetup.js:36]: who needs paragraphs anyway!

File: lib/components/Application.js [hunk 3]
```diff
@@ -83,44 +81,39 @@ export default class Application extends Component {
...
     firebase.database().ref(`users/${this.state.user.uid}/budgets/`).push({ itemValue, itemTime, budgetItem })
     firebase.database().ref(`users/${this.state.user.uid}/monthlyIncome`).push({ monthlyIncome })
-    this.setState({ budgets : { draftBudgetItem: '', draftItemValue: '', itemTime: moment(Date.now()).format('MM/D, h:mm a') } })
+    this.setState({ budgets: { draftBudgetItem: '', draftItemValue: '', itemTime: moment(Date.now()).format('MM/D, h:mm a') } })
   }
 
   render() {
     return (
-      <BrowserRouter>
+      <HashRouter basename="/public">
```
Comment (stephanieea) [inline lib/components/Application.js:89]: Is this what helps with reload?

**Review by stephanieea — APPROVED** [review]
🥇

File: lib/components/Application.js (hunk 3 above)
Comment (dylanavery720) [reply lib/components/Application.js:89]: yes @StephanieEA but in its own documentation its referred to as "hacky" lol so idk

**Comment by joshuajhun** [comment]
🔥

**Pull request merged**